StandardSocketsHttpHandler/Net/Http/SocketsHttpHandler/Http2Stream.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.

```

Comment:
You have removed the BOM from the beginning of the file, please revert this change

Reply:
Done 👍